target/linux/ramips/dts/OMEGA2.dts

```diff
@@ -0,0 +1,188 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "onion,omega", "mediatek,mt7628an-soc";
+	model = "Onion Omega2";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	bootstrap {
+		compatible = "mediatek,linkit";
+
+		status = "okay";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		system {
+			label = "onion:amber:system";
+			gpios = <&gpio1 12 1>;
+			default-state = "on";
```

Comment:
@greenbreakfast No. This is why you should use the dt-bindings. <&gpio1 12 1>; == <&gpio1 12 ACTIVE_LOW>

The LED works fine, though, it certainly is active-LOW. It's the reset-button you've got wrong.